doc/source/development/contributing.rst

```diff
@@ -837,6 +837,9 @@ to run its checks by running::
 
 without having to have done ``pre-commit install`` beforehand.
 
+Note that if you have conflicting installations of ``virtualenv``, then you may get an
```

Comment:
what is this for?

Reply:
`pre-commit` uses virtualenv - see the comment above (https://github.com/pandas-dev/pandas/pull/36826#issuecomment-703118027) for an error Daniel ran into